testing/bdd/features/timing/delay.feature

```diff
@@ -0,0 +1,125 @@
+    |            |              | 00:00:05  |          |          |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
+  # |            |              |           | 00:00:05 |          |          | 00:00:10 |                    |                  |                   | 00:00:15        |                 |               |            |          |          |
+    |            |              | 00:00:05  | 00:00:08 |          |          | 00:00:10 | 00:00:15           | 00:00:18         | 00:00:15          | 00:00:18        | 00:00:15        | 00:00:18      |            |          |          |
+    |            |              |           |          | 00:00:05 |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
+  # |            |              |           |          |          | 00:00:05 | 00:00:10 |                    |                  |                   |                 |                 | 00:00:15      |            |          |          |
+    |            |              |           |          | 00:00:05 | 00:00:08 | 00:00:10 | 00:00:15           | 00:00:18         | 00:00:15          | 00:00:18        | 00:00:15        | 00:00:18      |            |          |          |
+    | 00:00:05   | 00:00:20     | 00:00:03  |          |          |          | 00:00:05 | 00:00:10           | 00:00:25         | 00:00:13          | 00:00:25        | 00:00:13        | 00:00:25      |            |          |          |
+    | 00:00:05   | 00:00:20     | 00:00:03  |          | 00:00:04 |          | 00:00:05 | 00:00:10           | 00:00:25         | 00:00:13          | 00:00:25        | 00:00:17        | 00:00:25      |            |          |          |
+    | 00:00:10   | 00:00:10.500 |           |          | 00:00:03 | 00:00:06 | 01:00:00 | 01:00:10           | 01:00:10.500     | 01:00:13          | 01:00:10.500    | 01:00:13        | 01:00:10.500  |            |          |          |
+    |            |              | 00:00:10  | 00:00:20 | 00:00:05 | 00:00:07 | 00:00:01 | 00:00:11           | 00:00:21         | 00:00:11          | 00:00:21        | 00:00:16        | 00:00:18      |            |          |          |
+  # |            |              |           | 00:00:20 |          | 00:00:10 | 00:00:15 |                    |                  |                   | 00:00:35        |                 | 00:00:25      |            |          |          |
+  # |            |              | 00:00:20  |          |          | 00:00:10 | 00:00:15 | 00:00:35           |                  | 00:00:35          |                 | 00:00:35        | 00:00:10      |            |          |          |
```

Comment:
I'm looking again at this, and not sure why this last line is commented. The others are obviously there because we have not done #291 yet, but this seems to be a different case. Why does the following not apply?

* `updated_body_end` == `updated_div_end` == `updated_p_end` == `00:00:45`

These are the values I think I'd expect for computed times for these input values.